R/init.R

```diff
@@ -508,16 +508,59 @@ if (interactive() &&
         )
       }
 
+      print.help_files_with_topic <- function(h, ...) {
+        if (length(h) >= 1 && is.character(h)) {
+          file <- h[1]
+          path <- dirname(file)
+          dirpath <- dirname(path)
+          pkgname <- basename(dirpath)
+          requestPath <- paste0(
```

Comment:
This `requestPath` does not work under macOS.

When I run `?mean` in R terminal under macOS, nothing happens, and the following error appears in the debug console:

<img width="1245" alt="image" src="https://user-images.githubusercontent.com/4662568/98487807-2f422180-2260-11eb-957e-4a2e04e691a0.png">

Reply:
Did the recent pushes resolve this?

Reply:
Yes, it works quite nicely now!